File: solutions/uri/1013/1013.py
import sys

for line in sys.stdin:
    a, b, c = map(int, line.split())
    m = int((a + b + abs(a - b)) / 2)
    m = int((m + c + abs(m - c)) / 2)

    print(f"{m} eh o maior")

File: solutions/uri/1013/1013.clj
(ns main
  (:require [clojure.string :as str]))

(defn main []
  (loop [line (read-line)]
    (when line
      (let [[a b c] (->> (str/split line #" ")
                         (map #(Integer/parseInt %)))
            result (-> (- a b)
                       (Math/abs)
                       (+  b)
                       (+  a)
                       (/  2))
            result (-> (- c result)
                       (Math/abs)
                       (+  result)
                       (+  c)
                       (/  2))]
        (printf "%d eh o maior%n" result))
      (recur (read-line)))))

(main)
